spec/features/admin_edits_sauce_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+feature "when a user edits a sauce" do
+  scenario "user who is not an admin cannot see the 'Edit Sauce' link" do
+    user = FactoryBot.create(:user)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
```

Comment:
You should be able to use the Devise helper `login_as` to DRY up this process in each feature spec

Reply:
changed all applicable admin scenarios to use this helper